src/Umbraco.Infrastructure/BackgroundJobs/Jobs/LongRunningOperationsCleanupJob.cs

```diff
@@ -0,0 +1,53 @@
+using Umbraco.Cms.Core.Persistence.Repositories;
+using Umbraco.Cms.Core.Scoping;
+
+namespace Umbraco.Cms.Infrastructure.BackgroundJobs.Jobs;
+
+/// <summary>
+/// Cleans up long-running operations that have exceeded a specified age.
+/// </summary>
+public class LongRunningOperationsCleanupJob : IRecurringBackgroundJob
+{
+    private readonly ICoreScopeProvider _scopeProvider;
+    private readonly ILongRunningOperationRepository _longRunningOperationRepository;
+    private readonly TimeProvider _timeProvider;
+
+    private readonly TimeSpan _deleteTime = TimeSpan.FromDays(1);
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="LongRunningOperationsCleanupJob"/> class.
+    /// </summary>
+    /// <param name="scopeProvider">The scope provider for managing database transactions.</param>
+    /// <param name="longRunningOperationRepository">The repository for managing long-running operations.</param>
+    public LongRunningOperationsCleanupJob(
+        ICoreScopeProvider scopeProvider,
+        ILongRunningOperationRepository longRunningOperationRepository,
+        TimeProvider timeProvider)
```

Comment:
Given you have been thorough with the XML comments, just noting that `timeProvider` is missing for this one.

Reply:
Fixed!